src/queries/useConfigurations.test.js

```diff
@@ -73,10 +73,14 @@ describe('Given useConfigurations', () => {
         </QueryClientProvider>
       );
 
-      const { result, waitFor } = renderHook(() => useConfigurations({}), { wrapper });
+      const { result } = renderHook(() => useConfigurations({}), { wrapper });
 
+      // waitFor retries until it stops catching an error
+      // so we convert isLoading from being truthy to being errory
       await waitFor(() => {
-        return !result.current.isLoading;
+        if (result.current.isLoading) {
```

Comment:
Maybe it makes sense  move this part to separate function? I see it used in 4 places

```
if (result.current.isLoading) {
  throw new Error('Kaboom');
}
```

Reply:
🤷 I see your point, but there's not a lot going on here. I can imagine this also being written as
```
result.current.isLoading && throw new Error('Kaboom');
``` 
in which case we'd be replacing one line with ... one other line. It's a good thing to keep in mind though.